src/Pages/Review/components/BasketCard/BasketCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { removeFromBasket } from '../../actions';
+
+export class BasketCardComponent extends Component {
+    getCardCost (price, quantity) {
+        let cost = price;
+        cost = price * quantity;
+        return cost;
+    }
+
+    handleClick = (id) =>
+        this.props.dispatch(removeFromBasket(id));
+
+    render () {
+        const { item: { id, title, price, quantity } } = this.props;
+        return (
+            <div onClick={this.handleClick.bind(this, { id })}>
```

Comment:
Better to bind in the constructor to avoid performance anti-pattern. Please see https://reactjs.org/docs/optimizing-performance.html#examples